python-sdk/.test_durations

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
@dimberman are we committing this file intentionally?

Reply:
@tatiana yes that file is used by pytest-split to properly balance the tests

Reply:
Nice! ❤️